environment.yml

```diff
@@ -3,133 +3,169 @@ channels:
   - conda-forge
   - defaults
 dependencies:
```

Comment:
I use `conda env export --no-builds`. It would be more readable with just the main packages, but including all the dependencies used by libraries in the whistl environment ensures identical behavior between machines